.circleci/config.yml

```
@@ -886,6 +894,67 @@ jobs:
+                        export DOCKER_TAG_SUFFIX="-ee"
+                        export DOCKER_BUILD_ARGS="--build-arg GRAVITEEIO_VERSION=${GRAVITEEIO_VERSION} --build-arg GRAVITEEIO_DOWNLOAD_URL=https://download.gravitee.io/graviteeio-ee/apim/distributions --build-arg GRAVITEEIO_PACKAGE_NAME=graviteeio-ee-full"
+                      fi
+
+                      docker build ${DOCKER_BUILD_ARGS} --quiet -t graviteeio/apim-management-api:${GRAVITEEIO_VERSION_MAJOR}${DOCKER_TAG_SUFFIX} -t graviteeio/apim-management-api:${GRAVITEEIO_VERSION_MAJOR}.${GRAVITEEIO_VERSION_MINOR}${DOCKER_TAG_SUFFIX} -t graviteeio/apim-management-api:${GRAVITEEIO_VERSION}${DOCKER_TAG_SUFFIX} <<# parameters.docker_tag_as_latest >> -t graviteeio/apim-management-api:latest${DOCKER_TAG_SUFFIX} <</ parameters.docker_tag_as_latest >> ./gravitee-apim-rest-api/docker
+                      docker build ${DOCKER_BUILD_ARGS} --quiet -t graviteeio/apim-gateway:${GRAVITEEIO_VERSION_MAJOR}${DOCKER_TAG_SUFFIX}        -t graviteeio/apim-gateway:${GRAVITEEIO_VERSION_MAJOR}.${GRAVITEEIO_VERSION_MINOR}${DOCKER_TAG_SUFFIX}        -t graviteeio/apim-gateway:${GRAVITEEIO_VERSION}${DOCKER_TAG_SUFFIX}        <<# parameters.docker_tag_as_latest >> -t graviteeio/apim-gateway:latest${DOCKER_TAG_SUFFIX} <</ parameters.docker_tag_as_latest >>        ./gravitee-apim-gateway/docker
+                      docker build ${DOCKER_BUILD_ARGS} --quiet -t graviteeio/apim-management-ui:${GRAVITEEIO_VERSION_MAJOR}${DOCKER_TAG_SUFFIX}  -t graviteeio/apim-management-ui:${GRAVITEEIO_VERSION_MAJOR}.${GRAVITEEIO_VERSION_MINOR}${DOCKER_TAG_SUFFIX}  -t graviteeio/apim-management-ui:${GRAVITEEIO_VERSION}${DOCKER_TAG_SUFFIX}  <<# parameters.docker_tag_as_latest >> -t graviteeio/apim-management-ui:latest${DOCKER_TAG_SUFFIX} <</ parameters.docker_tag_as_latest >>  ./gravitee-apim-console-webui/docker        
+                      docker build ${DOCKER_BUILD_ARGS} --quiet -t graviteeio/apim-portal-ui:${GRAVITEEIO_VERSION_MAJOR}${DOCKER_TAG_SUFFIX}      -t graviteeio/apim-portal-ui:${GRAVITEEIO_VERSION_MAJOR}.${GRAVITEEIO_VERSION_MINOR}${DOCKER_TAG_SUFFIX}      -t graviteeio/apim-portal-ui:${GRAVITEEIO_VERSION}${DOCKER_TAG_SUFFIX}      <<# parameters.docker_tag_as_latest >> -t graviteeio/apim-portal-ui:latest${DOCKER_TAG_SUFFIX} <</ parameters.docker_tag_as_latest >>      ./gravitee-apim-portal-webui/docker
+
+                      docker image ls
+
+                      # Login to DockerHub
+                      docker login --username="${DOCKERHUB_BOT_USER_NAME}" -p="${DOCKERHUB_BOT_USER_TOKEN}"
+
+                      # Push all tags
+                      docker push graviteeio/apim-management-api:${GRAVITEEIO_VERSION} --all-tags
+                      docker push graviteeio/apim-gateway:${GRAVITEEIO_VERSION} --all-tags
+                      docker push graviteeio/apim-management-ui:${GRAVITEEIO_VERSION} --all-tags
+                      docker push graviteeio/apim-portal-ui:${GRAVITEEIO_VERSION} --all-tags
```

I think we can get rid of `GRAVITEEIO_VERSION`now as we are pushing all tags at once with `--all-tags`

```suggestion
                      docker push graviteeio/apim-management-api --all-tags
                      docker push graviteeio/apim-gateway --all-tags
                      docker push graviteeio/apim-management-ui --all-tags
                      docker push graviteeio/apim-portal-ui --all-tags
```

---

I removed `GRAVITEEIO_VERSION` and move the `--all-tags` before the image name in [`6495fe8` (#1403)](https://github.com/gravitee-io/gravitee-api-management/pull/1403/commits/6495fe84e89e2bd5364c5a7a24776ec652fdc624)